requirements.txt

```diff
@@ -0,0 +1,2 @@
+aiohttp==2.2.3
```

Comment:
Our requirements are for aiodocker developers, users install the library by `pip install aiodocker` without explicitly looking into requirement files at all.
Thus doesn't matter where we do place the file.
If we have everything in `requirements` folder -- let's put base requirements there.
`requirements/base.txt` sounds pretty good.